account_payment_mode/models/account_payment_method.py

```diff
@@ -22,6 +22,8 @@ class AccountPaymentMethod(models.Model):
     display_name = fields.Char(
         compute='compute_display_name',
         store=True, string='Display Name')
+    payment_mode_ids = fields.One2many(
+        'account.payment.mode', 'payment_method_id', 'Payment modes')
```

Comment:
I find it better to name these arguments

Reply:
@grindtildeath Have you seen my comment ?

Reply:
@adrienpeiffer Sorry I missed it, will fix it asap